src/utils/clients/lastfm/formatters/album-search-response.formatter.test.ts

```diff
@@ -0,0 +1,43 @@
+import { albumSearchFormatter } from './album-search-response.formatter';
+
+it('should format the results', () => {
+  const results = albumSearchFormatter({
+    data: {
+      results: {
+        albummatches: {
+          album: [
+            {
+              name: 'title',
+              artist: 'artist',
+              mbid: '1',
+              url: 'album.com',
```

Comment:
Use `example.com` or any actual url from a last.fm response

Reply:
(same for images)

Reply:
Fixed in e3d2218